sw/device/rom_exts/rom_ext_manifest_parser.h

```diff
@@ -232,6 +232,31 @@ bool rom_ext_get_public_key(rom_ext_manifest_t params,
 bool rom_ext_get_extension(rom_ext_manifest_t params, rom_ext_extension_id_t id,
                            rom_ext_extension_t *extension);
 
+/**
+ * Calculates the ROM_EXT interrupt vector value.
+ *
+ * The memory address where the ROM_EXT image interrupt vector resides, is
+ * relative.
```

Comment:
nit: Do you mean PC-relative?

Reply:
Is it not in the sense that it is relative to the slot (A or B) and perhaps the manifest size?

Reply:
This was copied over from other docs. 

All the returned values are absolute. It's that the field addresses end up being relative to the start of the ROM_EXT image. I will revisit the documentation in this module to clarify that.

Reply:
Removed this sentence, added "absolute" in other sentences where it was relevant.

Reply:
Thanks for the clarification, forgot to update this comment after checking the definitions and seeing that returned values are absolute.